test/src/api/commands/web-element/testIsVisible.js

```diff
@@ -122,7 +203,60 @@ describe('element().isVisible() command', function() {
     assert.strictEqual(elementId, '0');
   });
 
-  it('test .element().isVisible() assert', async function() {
+  it('test .element.find().isDisplayed() not displayed', async function () {
+    let elementId;
+
+    MockServer.addMock({
+      url: '/session/13521-10219-202/execute/sync',
+      method: 'POST',
+      response: JSON.stringify({
+        value: false
+      }),
+      onRequest(_, requestData) {
+        elementId = requestData.args[0]['element-6066-11e4-a52e-4f735466cecf'];
+      }
+    }, true);
+
+    const resultPromise = this.client.api.element.find('#signupSection').isVisible();
```

Comment:
you have written tests for `isVisible()` instead of `isDisplayed()`

Reply:
Fixed it